dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/PropertyUtils.java

```diff
@@ -255,4 +258,21 @@ public static void setValue(String key, String value) {
         properties.setProperty(key, value);
     }
 
+    public static Map<String, String> getPropertiesByPrefix(String prefix) {
+        if (Strings.isEmpty(prefix)) {
+            return null;
```

Comment:
Return 'null' in this method can trigger a NullPointException at line 98 of class file RegistryCenter.java